src/pages/bioclimatic/index.js

```diff
@@ -53,12 +54,15 @@ const Container = () => {
     scenarios &&
     scenarios.reduce((acc, sc) => {
       const years = getYears(sc);
+      const startIndex = years.indexOf(startYear * 1);
+      const endIndex = years.indexOf(endYear * 1);
```

Comment:
`* 1` ?

Reply:
It was my "creative" way of converting params in URL from string to number - already changed to `years.indexOf(Number(startYear))`  thanks :+1: